docs/usage.rst

```diff
@@ -1,22 +1,78 @@
 Usage Examples
 ==============
 
-Encoding & Decoding Tokens
+Encoding & Decoding HS256 Tokens
 ---------------------------------
 
 .. code-block:: python
 
-    >>import jwt
-    >>encoded = jwt.encode({'some': 'payload'}, 'secret', algorithm='HS256')
+    >> import jwt
+    >> encoded = jwt.encode({'some': 'payload'}, 'secret', algorithm='HS256')
+    >> encoded
     'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzb21lIjoicGF5bG9hZCJ9.4twFt5NiznN84AWoo1d7KO1T_yoc0Z6XOpOVswacPZg'
+    >> jwt.decode(encoded, 'secret', algorithms=['HS256'])
+    {u'some': u'payload'}
+
+
+Encoding & Decoding RS256 Tokens
+---------------------------------
+
+.. code-block:: python
+
+    >> import jwt
+    >> from cryptography.hazmat.backends import default_backend
+    >> from cryptography.hazmat.primitives import serialization
+    >> # openssl genrsa 1024 | tee private_key.pem
+    >> private_key_pem = '''
+    .. -----BEGIN RSA PRIVATE KEY-----
+    .. MIICXgIBAAKBgQDBCeVu627zFZ1JH9/Wi/J/bs6zC3bUFl0ASfE6XHGxyPTAPXgJ
+    .. nc7AsnRBxbNA692v1srkZr1X1BwUbzcaMRZwpGi4vO4VwzLldJC/YLFp5z6C66bg
+    .. GvRrp5pQhu4ntuHR82yS2X/IBsmMArUug9mO/LyoGthqRBVic/a9l9+INQIDAQAB
+    .. AoGBAIekj45waubuwjXW6u+UKRL4ZtAS9y2yhSklzBbpTI7TmX/X8Zg4RkbLXru0
+    .. 0u+EjaL4eFskAlpL1mtZdsu1wICvyiFKuvh5WE+OwxBLpju/7AuZ9lCan9HR0X8P
+    .. EXASwU8ZFGTbLWPJePeiWl41431EAZtq/cWDSB/RQeoa2mNBAkEA5OKP+uxjSH1o
+    .. kCg+YqmlaakQ+2b8fS5J0ZyriVmoOAG0af647rsf4G3x3tXokoLXLn/620DE1HQ5
+    .. fCqI7l2xhQJBANfoNwSAqMbrURS2g4X1F6t5kxqPW7QNYrqPiAwGeXrJlG0Y8U5v
+    .. Yv73vRlnigdSJzTQOnY0FhniFWuScIdk4vECQG6iTLIfHQZnB+nWagFKuxfNjtXW
+    .. O+lOPIRDVG75lWQs/sXVSBKtBIV431a00swuzlA9sEXWks2WuEqaTMHbK/kCQQCG
+    .. 3uV3Z5OG5zp4EOcqCAeoM0LERadIW1BAMCcRM/4wyLlySTF8CLKziThUJUyg9B3P
+    .. rP/IFRN1SbiNwSWQPmJRAkEAlBTPJeblIe0u/Zj8f4AKaSnKQXBvTwA3OP/FyjHR
+    .. 8KaS4wpfYQvYuXZe6EDB6d6AxNxDuYn5T6D/qtVAR/eZWw==
+    .. -----END RSA PRIVATE KEY-----
```

Comment:
We definitely should not be showing a complete RSA key pair in the example. This will no doubt result in somebody using these keys in production which would be terrible. I'd definitely truncate this to something like:

```
>> private_key_pem = '''
.. -----BEGIN RSA PRIVATE KEY-----
.. MIICXgIBAAKBgQDBCeVu627zFZ1JH9/Wi/J/bs6zC3bUFl0ASfE6XHGxyPTAPXgJ
   ...
.. -----END RSA PRIVATE KEY-----